src/plugins/lab.ts

```diff
@@ -1,6 +1,7 @@
-import { LabaColor } from "../types";
+import { LabaColor, AnyColor } from "../types";
 import { Plugin } from "../extend";
 import { parseLaba, roundLaba, rgbaToLaba } from "../colorModels/lab";
+import { deltaE00 } from "../get/getPerceivedDifference";
```

Comment:
```js
import { getPerceivedDifference } from "../get/getPerceivedDifference";
```
or
```js
import { getDeltaE } from "../get/getDeltaE";
```

Reply:
Should I name it `getDeltaE00` instead? To be clear what version of the algorithm we provide?

Reply:
Let's use `getDeltaE00` name internally 👍